src/components/DialogInStepperWrapper/DialogInStepperWrapper.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react'
+
+import { DialogModalContext, StepperContext } from '../Contexts'
+
+const DialogInStepperWrapper = () => {
+  const { currentPages } = useContext(DialogModalContext)
+  const { currentStep } = useContext(StepperContext)
```

Comment:
Is it possible to homogenise those variables names? Relation between pages and steps is not obvious until line 11 is read.

Reply:
Aurais-tu des noms en tête ? Si je change juste `currentStep` par `activeStep`, ça te semble mieux ?
Pour moi les noms actuels sont très clairs, le 'current' peut faire un peu répétition si on lit vite d'où peut-être changer le nom du step par `activeStep`

Reply:
Ma remarque était plus sur le Step vs Pages. D'un côté on manipule des `Steps`, et de l'autre on manipule des `Pages` alors qu'en lisant le code j'ai l'impression qu'il s'agit des mêmes concepts.

Reply:
En faite `currentPages` est une liste d'objets qui va renseigner le contenu de chaque "modèle de page" du processus de création d'un Paper.
Ces "modèles" seront des composants qui ce chargeront dynamiquement avec ces infos. (pas encore implémenté)
Le `stepper` lui va permettre de naviguer entre ces pages.
On est bien sur 2 choses distinctes.

Voila j'espère avoir été clair et surtout répondu juste 😁